clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/utils/ClientUtils.kt

```diff
@@ -2,36 +2,24 @@ package org.utbot.cpp.clion.plugin.utils
 
 import com.intellij.openapi.actionSystem.AnActionEvent
 import com.intellij.openapi.actionSystem.CommonDataKeys
-import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.components.service
 import com.intellij.openapi.project.Project
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.job
 import org.utbot.cpp.clion.plugin.client.Client
 import org.utbot.cpp.clion.plugin.client.ClientManager
 import org.utbot.cpp.clion.plugin.client.logger.ClientLogger
 
-val Project.logger: ClientLogger
-    get() = this.service()
+//TODO: remove logger from this file
```

Comment:
maybe you have some comments why it should be removed or some ideas where it should be. The logger is accessed from different places (requests, handlers, some UI components) to show logs to the user

Reply:
I hope that we will be able to implement logger without services similarly to UtBot Java approach